package.json

```diff
@@ -8,19 +8,22 @@
   "repository": "github:nextstrain/nextstrain.org",
   "homepage": "https://nextstrain.org",
   "engines": {
-    "node": "^16",
-    "npm": "^7 || ^8"
+    "node": "^20",
+    "npm": "^10"
   },
   "scripts": {
     "build": "./build.sh",
-    "lint": "eslint --ext .js,.jsx .",
+    "lint": "npm run lint:server",
+    "lint:server": "DEBUG=eslint:cli-engine npx eslint --ext .js,.jsx .",
+    "lint:static-site": "cd static-site && DEBUG=eslint:cli-engine npx eslint --ext .js,.jsx .",
     "server": "node server.js",
     "groups": "node server.js --app ./src/groupsApp.js",
     "start": "npm run server",
     "//test": "echo 'NODE_OPTIONS is required until this issue is resolved: https://github.com/facebook/jest/issues/9430 '",
     "test": "NODE_OPTIONS='--experimental-vm-modules' jest --roots ./test/",
-    "test:ci": "start-server-and-test 'RESOURCE_INDEX=test/date_descriptor_index.json node server.js --verbose >./test/server.log 2>&1' http://localhost:5000 test",
-    "dev": "./develop.sh"
+    "test:ci": "next build static-site && start-server-and-test 'NEXT_JS_PRODUCTION=1 RESOURCE_INDEX=test/date_descriptor_index.json node server.js --verbose >./test/server.log 2>&1' http://localhost:5000 test",
```

Comment:
The (GitHub actions) test `test/cryptography.test.js` has been consistently failing. I'll do some digging as obviously this is blocking. I don't think anything in this PR is at all related to the code paths it uses, except for the maybe the node upgrade, but I cannot recreated this failure locally (and it's not dependent on any AWS environment variables as far as I can tell).

Reply:
`cryptography.test.js` passes on nodejs 20.9.0 (which is what I was developing on, as it's the latest available in conda) but consistently failed when [run on 20.12.0](https://github.com/nextstrain/nextstrain.org/actions/runs/8566685893/job/23476917786#step:11:257) and [20.11.1](https://github.com/nextstrain/nextstrain.org/actions/runs/8564336483/job/23470717176) in GitHub Actions. Pinning GitHub actions to 20.9.0 allowed the [test to pass](https://github.com/nextstrain/nextstrain.org/actions/runs/8575901351/job/23505846732).

For the purposes of this PR pinning to 20.9.0 is fine and I'll create a separate issue to track this bug.

Separately, while debugging this I identified that the version we've elected to use for CI (`actions/setup-node`) is slightly different from the version Heroku will use (via `package.json`). I'll either fix this here or create a separate issue.

Reply:
> I'll either fix this here or create a separate issue.

I've done this, and as expected GitHub actions is using 20.9.0 and [tests pass](https://github.com/nextstrain/nextstrain.org/actions/runs/8583144975/job/23522029992). I'll double check the Heroku review app is also pulling in 20.9.0, but I'd be very surprised if it's not.

Reply:
> I'll double check the Heroku review app is also pulling in 20.9.0, but I'd be very surprised if it's not.

Yes, it is.

Reply:
https://github.com/nextstrain/nextstrain.org/issues/814